tsconfig.json

```diff
@@ -1,14 +1,15 @@
 {
   "compilerOptions": {
     "baseUrl": ".",
-    "module": "NodeNext",
-    "moduleResolution": "NodeNext",
+    "module": "esnext",
```

Comment:
The config have been recently updated to fix the problem with deployment to Vercel
https://github.com/akellbl4/spotify-badge/commit/f82cbcfb42bad06021244b11a957e2e2d77184bd

Reply:
okay, for some reason this set up + downgraded node modules allow me not to have these type errors, any suggestions ? what might be wrong with my setup ?
<img width="1136" alt="Screenshot 2024-11-13 at 20 26 22" src="https://github.com/user-attachments/assets/239178f6-4327-4852-a01b-3a92dde75023">

Reply:
<img width="1170" alt="Screenshot 2024-11-13 at 20 27 45" src="https://github.com/user-attachments/assets/7f877ed2-cfaf-4eca-96e7-a55c2908927b">

Reply:
let me check